src/mlpack/methods/pca/decomposition_policies/randomized_svd_method.hpp

```diff
@@ -18,12 +18,11 @@
 #include <mlpack/methods/randomized_svd/randomized_svd.hpp>
 
 namespace mlpack {
-namespace pca {
 
 /**
  * Implementation of the randomized SVD policy.
  */
-class RandomizedSVDPolicy
+class RandomizedSVDPCAPolicy
```

Comment:
Agreed.  I wonder if it might be better to eventually refactor `PCA` so that the decomposition policy happens to already match the methods provided by `RandomizedSVD`, etc; then, this "shim" class won't be needed.